README.md

```diff
@@ -1,22 +1,23 @@
 # BladeDISC Introduction
 
-* [Overview](#overview)
-  + [Features and Roadmap](#features-and-roadmap)
-    - [Frontend Framework Support Matrix](#frontend-framework-support-matrix)
-    - [Backend Support Matrix](#backend-support-matrix)
-    - [Deployment Solutions](#deployment-solutions)
-  + [Numbers of Typical Workloads](#numbers-of-typical-workloads)
-    - [Advantage in Dynamic Shape Workloads](#advantage-in-dynamic-shape-workloads)
-* [API QuickView](#api-quickview)
-  + [For TensorFlow Users](#for-tensorflow-users)
-  + [For PyTorch Users](#for-pytorch-users)
-* [Setup and Examples](#setup-and-examples)
-* [Publications](#publications)
-* [Tutorials and Documents for Developers](#tutorials-and-documents-for-developers)
-* [How to Contribute](#how-to-contribute)
-* [FAQ](#faq)
-  + [Roadmap with mlir-hlo Project](#roadmap-with-mlir-hlo-project)
-* [Contact Us](#contact-us)
+- [BladeDISC Introduction](#bladedisc-introduction)
```

Comment:
Why do we want to include the index of the upper-level title inside the upper-level block.

Reply:
It's auto-generated by vs-code.

Reply:
Done, thanks!